src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/ResidentJudgeFtpaDecisionConfirmationTest.java

```diff
@@ -5,9 +5,7 @@
 import static org.hamcrest.Matchers.containsString;
 import static org.junit.Assert.*;
 import static org.mockito.Mockito.*;
-import static uk.gov.hmcts.reform.iacaseapi.domain.entities.AsylumCaseFieldDefinition.FTPA_APPELLANT_RJ_DECISION_OUTCOME_TYPE;
-import static uk.gov.hmcts.reform.iacaseapi.domain.entities.AsylumCaseFieldDefinition.FTPA_APPLICANT_TYPE;
-import static uk.gov.hmcts.reform.iacaseapi.domain.entities.AsylumCaseFieldDefinition.UPLOAD_HOME_OFFICE_BUNDLE_ACTION_AVAILABLE;
+import static uk.gov.hmcts.reform.iacaseapi.domain.entities.AsylumCaseFieldDefinition.*;
```

Comment:
AsylumCaseFieldDefinition.*
Explicit definition is suggested.

Reply:
case-api does allow this formatting. May be it is a wider thing to be consistent across.